scripts/build-debianpackage

```diff
@@ -27,29 +27,57 @@ TOP_BUILDDIR="$HOME/debbuild/packaging"
 mkdir -p "$TOP_BUILDDIR"
 rm -rf "${TOP_BUILDDIR:?}/${PKG_NAME}"
 mkdir -p "${TOP_BUILDDIR}/${PKG_NAME}"
+# Move changelog into place (we have separate changelogs for each platform)
+PLATFORM="$(lsb_release -sc)"
 
 # Validate required args.
 if [[ -z "${PKG_NAME:-}" ]]; then
     echo "Set PKG_NAME of the build";
     exit 1
 fi
 
+
+function find_latest_version() {
```

Comment:
this is a good idea. will your change still support supplied version numbers in case we're building a dev release?

Reply:
Yes, fully backwards compatible in that PKG_VERSION & PKG_PATH can still be provided as overrides, they're just not _required_. The verification logic needs some work before it's ready for full review—at least:

1. use a dedicated keyring so that _only_ the prod signing key is used to validate tags
2. detect if an "rc" tag is being used, and if so, don't expect a prod sig

Then it should be ready to bang around on.